src/recipes/IngredientEdit.js

```diff
@@ -0,0 +1,18 @@
+export default function IngredientEdit(props) {
```

Comment:
FYI: I think the pattern we use here most commonly would be `EditIngredient`, etc...

Reply:
Aha got it, thank you 👍